src/test/groovy/com/github/vkorobkov/jfixtures/loader/RowTest.groovy

```diff
@@ -94,4 +94,70 @@ class RowTest extends Specification {
         expect:
         EqualsVerifier.forClass(Row).verify()
     }
+
+    def "#columns(Object...) returns row from key/value pairs"() {
+        given:
+        def row = new Row("vlad", columns)
+
+        when:
+        def result = row.columns(
+                "name", "Vlad",
+                "age", 30,
+                "hobby", "sleep"
+        )
+
+        then:
+        with(result.columns){
+            size() == 4
+            toMapString() == "[id:Value(value=1, type=AUTO), name:Value(value=Vlad, type=TEXT), age:Value(value=30, type=AUTO), hobby:Value(value=sleep, type=TEXT)]"
```

Comment:
Maybe it makes sense to instantiate "expected" map and call `toMapString` of it?
```java
toMapString() == [id: Value.of(1), ....].toMapString()
```

Reply:
:+1: